### PR TITLE
#88 - Search bar does not work

### PR DESCRIPTION
**Pull request by daceynolan**
Refactor intercom, to allow the search bar at the top of the page to work as expected. 

https://www.loom.com/share/8958bcbfaba64bed8ed44ba2b658a540

closes https://github.com/triggermesh/docs/issues/88

### CODE REVIEW

File: docs/extra.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+window.intercomSettings = {
+        app_id: "itolwzo2"
+    };
+
+    (function () { var w = window; var ic = w.Intercom; if (typeof ic === "function") { ic('reattach_activator'); ic('update', w.intercomSettings); } else { var d = document; var i = function () { i.c(arguments); }; i.q = []; i.c = function (args) { i.q.push(args); }; w.Intercom = i; var l = function () { var s = d.createElement('script'); s.type = 'text/javascript'; s.async = true; s.src = 'https://widget.intercom.io/widget/' + "itolwzo2"; var x = d.getElementsByTagName('script')[0]; x.parentNode.insertBefore(s, x); }; if (w.attachEvent) { w.attachEvent('onload', l); } else { w.addEventListener('load', l, false); } } })();
```
Comment (antoineco) [inline docs/extra.js:5]: nit: GitHub is reporting a missing new line at the end of the file. Mis-configured IDE, maybe?

File: mkdocs.yml [hunk 2]
```diff
@@ -22,7 +25,7 @@ nav:
     - Cloud: https://cloud.triggermesh.io
     - About: about.md
 theme:
-  name: mkdocs
+  name: "mkdocs"
```
Comment (antoineco) [inline mkdocs.yml:28]: Those quotes aren't necessary.

**Review by antoineco — APPROVED** [review]
Approving with 2 nits 👍

**Review by cab105 — APPROVED** [review]

File: mkdocs.yml (hunk 2 above)
Comment (daceynolan) [reply mkdocs.yml:28]: Haha I thought so, but their docs showed it so I got confused lol

File: docs/extra.js (hunk 1 above)
Comment (antoineco) [reply docs/extra.js:5]: @daceynolan in case you use VSCode: https://stackoverflow.com/a/44704969/4716370

Comment (antoineco) [reply docs/extra.js:5]: To be clear, I'm talking about this here:
image

**Pull request merged**